test/integration/staking-rewards.ts

```diff
@@ -64,8 +64,17 @@ async function initialize() {
   bdEu = await getBdEu(hre);
   bdx = await getBdx(hre);
 
-  stakingRewards_BDEU_WETH = await getBDStableWethStakingRewards(hre, await bdEu.symbol());
-  stakingRewards_BDEU_WBTC = await getBDStableWbtcStakingRewards(hre, await bdEu.symbol());
+  let contract = await getStakingRewardsWithWeth(hre, await bdEu.symbol());
+  if (!contract) {
+    throw new Error(`StakingRewads contract BDEU-WETH doesn't exist`);
+  }
+  stakingRewards_BDEU_WETH = contract;
+
+  contract = await getStakingRewardsWithWbtc(hre, await bdEu.symbol());
```

Comment:
NIT I think it's not the best practice to reuse variables for different purposes.